Dockerfile

```diff
@@ -65,3 +53,6 @@ RUN if [ "${DB}" = "oracle" ]; then unzip /opt/oracle/instantclient-basiclite-li
  && (cd /opt/oracle/instantclient_12_2/ && ln -s libclntsh.so.12.1 libclntsh.so) \
  && rm -rf /opt/system/vendor/oracle \
  && rm -rf /opt/oracle/*.zip; fi
+
+USER 1001
+RUN bash -c "bundle install && npm install"
```

Comment:
Probably installing the gem as root but meaning it should know scl-enable stuffs ... which IMO is not the right thing to do.